src/probe.rs

```diff
@@ -5,6 +5,10 @@ use probe_rs::{DebugProbeInfo, Probe};
 
 use crate::cli;
 
+const NO_PROBE_FOUND_ERR: &str = "no probe was found.\n
+Common reasons for this are faulty cables or missing permissions.
+For detailed instructions, visit: https://github.com/knurling-rs/probe-run/tree/2f138c3#troubleshooting";
```

Comment:
Can't we link to the main branch troubleshooting section?

```suggestion
For detailed instructions, visit: https://github.com/knurling-rs/probe-run#troubleshooting";
```

Reply:
Just saw the discussion in the issue